cmd/node/node.go

```diff
@@ -444,22 +432,29 @@ func (app *SpacemeshApp) Start(cmd *cobra.Command, args []string) {
 		log.Panic("Could not retrieve identity err=%v", err)
 	}
 
+	vrfPublicKey, vrfPrivateKey, err := crypto.GenerateVRFKeys()
```

Comment:
That is a mock? if so add a comment. If not what is it? we still haven't integrated bls...

Reply:
this is not a mock. it uses some signature scheme